src/common_types/container_types.hpp

```diff
@@ -288,7 +288,7 @@ struct checksum_key {
 static_assert(sizeof(checksum_key) == sizeof(checksum_key::key));
 
 struct player_name {
-	char data[8];
+	char data[8] = { 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
can't even fit "schombert" in that ... plus it will need even more room when it supports unicode